CMakeLists.txt

```diff
@@ -111,9 +112,11 @@ target_sources(suit PRIVATE
   src/suit_directive.c
   src/suit.c
   )
-target_include_directories(suit PUBLIC
+
+target_include_directories(suit_interface INTERFACE
   include
   )
+target_link_libraries(suit PUBLIC suit_interface)
```

Comment:
It's good to have this target!
Please take a look at `ncs/CMakeLists.txt`, if anything there should be added/can be simplified :slightly_smiling_face: